app/controllers/movies_controller.rb

```diff
@@ -15,7 +15,7 @@ def index
   def show
     @user = User.find(params[:user_id])
     @movie_facade = DetailedMovieFacade.new(params[:id])
-    @cast = MovieService.get_movie_cast(params[:id]).first(10)
-    @reviews = MovieService.get_movie_reviews(params[:id])
+    @cast_facade = MovieCastFacade.new(params[:id])
```

Comment:
We'll want to call for the first 10 at some point in the code, and also test to make sure that only the first 10 cast members are being returned